### PR TITLE
Add site tab favicon

### PR DESCRIPTION
**Pull request by bob-zs**
Feel free to let me know if you want a different favicon if you think it shouldn't be bubble tea.

Other places to find favicons:
https://www.flaticon.com/free-icons/bubble-tea
https://favicon.io/emoji-favicons/bubble-tea/

-----
What it shows in dev:
Screen Shot 2022-08-26 at 11 33 04 AM

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
 const express = require('express');
+const favicon = require('serve-favicon')
+const path = require('path');
+
 const app = express();
 app.use(express.json());
-const { User } = require('./db');
-const path = require('path');
```
Comment (bob-zs) [inline app.js:5]: Adding favicon by express

File: app.js [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+app.use('/favicon_io', express.static('favicon_io'));
 
-app.get('/', (req, res)=> res.sendFile(path.join(__dirname, 'index.html')));
+app.get('/', (_req, res)=> res.sendFile(path.join(__dirname, 'index.html')));
 
 app.use('/api/orders', require('./routes/orders'));
 app.use('/api/sessions', require('./routes/sessions'));
 app.use('/api/products', require('./routes/products'));
 
-app.use((err, req, res, next)=> {
```
Comment (bob-zs) [inline app.js:15]: Just extra cleanup. Unused parameters are prefixed with a `_`.

File: favicon_io/about.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+This favicon was generated using the following graphics from Twitter Twemoji:
+
+- Graphics Title: 1f9cb.svg
+- Graphics Author: Copyright 2020 Twitter, Inc and other contributors (https://github.com/twitter/twemoji)
+- Graphics Source: https://github.com/twitter/twemoji/blob/master/assets/svg/1f9cb.svg
+- Graphics License: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
```
Comment (bob-zs) [inline favicon_io/about.txt:6]: Leaving the link to creative commons license.

File: app.js [hunk 4]
```diff
@@ -1,18 +1,22 @@
 const express = require('express');
+const favicon = require('serve-favicon')
+const path = require('path');
+
 const app = express();
 app.use(express.json());
-const { User } = require('./db');
-const path = require('path');
+
+app.use(favicon(path.join(__dirname, 'favicon_io', 'favicon.ico')))
```
Comment (bob-zs) [inline app.js:9]: Adding favicon by express

**Review by alejandro-loja — APPROVED** [review]
Nice favicon choice

**Pull request merged**